README.md

```diff
@@ -46,6 +46,7 @@ _**Note:** The behavior of actions like this one is currently limited in the con
   - [Mypy](https://mypy.readthedocs.io/)
   - [oitnb](https://pypi.org/project/oitnb/)
   - [Pylint](https://pylint.pycqa.org)
+  - [Ruff](https://ruff.rs)
```

Comment:
@charliermarsh is it safe to assume https://ruff.rs to be the website? Currently it redirects to the github repository.

Reply:
:D Thanks for supporting Ruff! I haven't read through the code here but let me know if you have any questions.